Rent-Cap/website
# Existing rent control check

This matches the `coverage/existing-rent-control-check` page to the mockup

To test local,
`git pull`

`git checkout existing-rent-control-check`

`npm run develop`

open browser to: http://localhost:8000/coverage/existing-rent-control-check

compare to Question 1 mockup on figma: 
https://www.figma.com/file/VVJ6GIMBJajFLRJDROFDXm/Rent-Cap?node-id=1%3A106
